SIL.Core.Tests/IO/PathUtilitiesTests.cs

```diff
@@ -323,6 +323,17 @@ public void OpenDirectoryInExplorer_PathIsADirectoryHasCombiningCharacters_Still
 			}
 			PathUtilities.OpenDirectoryInExplorer(path);
 		}
+
+		[TestCase(null, Result = false)]
+		[TestCase("", Result = false)]
+		[TestCase("directory", Result = true)]
```

Comment:
The only "true" test here, "directory", is ambiguous because it appears non-final and final, so it doesn't prove either works, just that one of them works.